packages/benchmarks/benches/bench_load_wrapper.rs

```diff
@@ -0,0 +1,79 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use criterion::BatchSize::PerIteration;
+
+use polywrap_client::client::PolywrapClient;
+use polywrap_core::{
+    uri::Uri,
+    wrap_loader::WrapLoader,
+};
+use polywrap_client_builder::types::{ClientConfigHandler};
+use polywrap_core::uri_resolver_handler::UriResolverHandler;
+use polywrap_benchmarks::{get_fibonacci_dir};
+
+fn prepare_client() -> PolywrapClient {
+    let builder = polywrap_client_default_config::build();
+    let config = builder.build();
+    PolywrapClient::new(config)
+}
+
+pub struct UriCase {
+    pub id: String,
+    pub uri: Uri,
+}
+
+pub fn prepare_uris() -> Vec<UriCase> {
+    let fs_uri = UriCase {
+        id: "fs_uri".to_string(),
+        uri: Uri::try_from(format!("fs/{}", get_fibonacci_dir("rs"))).unwrap(),
+    };
+    let http_uri = UriCase {
+        id: "http_uri".to_string(),
+        uri: Uri::try_from(format!(
+            "http/https://raw.githubusercontent.com/polywrap/rust-client/kris/debug-slow-invocation/packages/benchmarks/fibonacci/rs/build"
```

Comment:
I think we should create a `benchmark-wraps` repo in the polywrap organization and use that

Reply:
yeah, sounds good! I'll do that soon.